pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/design/xml/TooManyFields.xml

```diff
@@ -63,33 +63,12 @@ public class Foo {
 	Integer a5; 
 	Integer a6; 
 	public class Bar {
-		Integer a7; 
-		Integer a8; 
-		Integer a9; 
-		Integer a10; 
-		Integer a11; 
-		Integer a12; 
-	}
-}
-		]]></code>
-    </test-code>
-    
-    <test-code>
-        <description>outer class, inner interface, both OK</description>
```

Comment:
Apex also probably doesn't support inner interfaces, right?

Reply:
In fact it does, but it only supports static fields in outer classes, and interfaces cannot have fields AFAIK